storage-manager/src/smls.cpp

```diff
@@ -80,7 +82,11 @@ void lsOffline(const char *path)
             }
             else
                 cout.width(15);
-            cout << right << _stat.st_size << left << " " << entry << endl;
+
+            struct tm *my_tm = localtime(&_stat.st_mtim.tv_sec);
+            char date[32];
```

Comment:
localization possibilities may need more space...?  I'd say pad this to 100 bytes to eliminate the question b/c who cares.